androidbrowserhelper/build.gradle

```diff
@@ -43,13 +43,14 @@ android {
 }
 
 dependencies {
-    api 'androidx.browser:browser:1.2.0'
+    api 'androidx.browser:browser:1.3.0-alpha03'
 
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
     implementation 'androidx.annotation:annotation:1.1.0'
     implementation 'androidx.appcompat:appcompat:1.0.2'
     implementation 'androidx.core:core:1.0.2'
+    implementation 'com.google.android.gms:play-services-location:17.0.0'
```

Comment:
Is there any way to make this optional? I'm wary of increasing the APK size for users who don't use this feature?

Reply:
The only way I can think of is make it default to use the android location provider, and add another target in the demos/ that include the gms provider. 
The down side is in Chrome tab, default is using the gms one, and fallback to the android one if no play service available. So this may make TWA behaviour different with Chrome.
WDYT?

Reply:
So, I don't think we'd add something in demos, maybe top level (like `playbilling`) or maybe in some `features` directory or something. Then the TWA could depend on just the core `android-browser-helper` or add in the features it wants.

Ideally this core part would have no code to do with location delegation at all (or maybe just some stubs), and we would find some way to register them.

I'm thinking something like this:

1. The `DelegationService` has an interface like `ExtraCommandHandlers`.
1. The `DelegationService` has a list of these `ExtraCommandHandlers` and when it gets an extraCommand, it loops through them, seeing if any can deal with it.
1. You move all the code you added to `DelegationService` into the `LocationDelegationExtraCommandHandler`, which lives in a different module.
1. Somehow, the LocationDelegation module registers itself to the `DelegationService`.

That way we keep the code `android-browser-helper` module light and we can develop all these new feature modules in quite a neat, separate way. The only problem is that I haven't quite figured out how to do step 4 yet.

Reply:
I updated the PR to using `ExtraCommandHandler` and a separate `locationdelegation` module. Will add a demo and the update the bubblewrap change later